src/sources/host_metrics.rs

```diff
@@ -751,10 +748,6 @@ pub fn init_roots() {
 }
 
 impl FilterList {
-    fn is_empty(&self) -> bool {
```

Comment:
Looks like this is still needed for tests.

Reply:
Ah yes it is. Thanks!